docs/src/trouble/index.md

```diff
@@ -0,0 +1,104 @@
+# {#ref-ts} Troubleshooting
+
+If you receive one of the following errors: 
+
+```cmd
+reach: /app/tmp/out.sh: openFile: permission denied (Permission denied)
+```
+
+OR
+
+```
+chmod: cannot access '/tmp/reach.../out.sh': No such file or directory
+./reach: 1: /tmp/reach.../out.sh: not found
+```
+
+* This generally means that Reach was previously run or installed with `sudo` and directories were created that only `root` can access. 
+
+* If this is the case, delete the local copy of the Reach program and reinstall per the instructions.
+
+* If use of `sudo` during installation is not the cause then try [updating Docker](https://techdirectarchive.com/2021/10/17/how-to-manually-update-docker-desktop/) and Docker-compose.
+
+"I'm running Windows 8 and cannot install Docker."
+
+Unfortunately, Docker Desktop does not support Windows 8.
+Also, Reach is not able to run on Windows 8. 
+
+Possible solutions are:
+* Upgrade to Windows 10 or Windows 11.
+* Partition the drive (or use a second drive) and install of a copy of a supported Linux distro.
+* Run a VM with a supported OS.
+It is likely that this option has the worst experience due to the splitting of system resources.
+
+Executing `./reach version` on Linux outputs `./reach: Is a directory`
+
+* This error usually occurs when `reach` is being called from a directory where it is not installed.
+Compare where Reach is installed with the current working directory. 
+* Check the current working directory with the terminal command `pwd`.
+
+If running `./reach run` requires `sudo`, then it is likely that:
+
+* A component used by `reach` was installed/created using `sudo`, such as Docker or the folder directory.
+Make sure that all components are installed or created without using super-user permissions. 
+
+* Docker might have been installed using Snap.
+Snap installation in Ubuntu creates permissions issues.
+Uninstall Docker and then re-install using the `apt` command in the terminal.
+
+## {#ref-ts-config} Reach configuration and REACH_CONNECTOR_MODE
+
+`Missing REACH_CONNECTOR_MODE`
+
+* Reach expects to be told to connect to a specific consensus network.
+Running ONE of the following commands should fix this error:
+
+`export REACH_CONNECTOR_MODE=ALGO`
+
+`export REACH_CONNECTOR_MODE=ETH`
+
+`export REACH_CONNECTOR_MODE=CFX`
+
+When executing `./reach config`, the compiler prints the following message:
```

Comment:
I appreciate what this section is trying to do given the surprising amount of confusion we've seen but I think you should actually just remove it:

- This isn't a sign of trouble
- The compiler isn't relevant here; I think you mean CLI
- It won't always be `~/.profile` - it depends on the user's `$SHELL` and how they've configured it, e.g. https://github.com/reach-sh/reach-lang/blob/8f9ceada61a0bf3a83d4de981014d4eb15f64c9e/hs/app/reach/Main.hs#L1708
- Right now I'm trying to add more padding around how `reach config` works anyway, so I think you should let me fail at that until I discover the least-failable way and we take this back out of confused users' hands